docs/faq.rst

```diff
@@ -13,18 +13,73 @@ Frequently asked questions.
 Setting up environment
 ----------------------
 
+**Q: jupyter notebook/lab cannot find qiskit_metal. Why is that?**
 
-**Why do I have an inactive developer path on MacOs?**
+**A:** If you are seeing: ``ModuleNotFoundError: No module named 'qiskit_metal'`` in jupyter notebook/lab, you are using a jupyter installation outside of your current environment and you therefore need to create a kernel that refers to the environment where you installed qiskit_metal. To do so, install and configure ipykernel.
```

Comment:
Could we add a picture like this?
![image](https://user-images.githubusercontent.com/15937254/121240194-10dcec80-c868-11eb-847c-9177717ed05f.png)

See https://doc.cocalc.com/howto/jupyter-kernel-selection.html

Reply:
done. merging